### PR TITLE
Add Swagger-ui to AdvantEDGE platform

### PR DESCRIPTION
**Pull request by pastorsx**
Add Swagger ui to AdvantEDGE platform:

Swagger ui interface was added to the AdvantEDGE platform that shows all the apis* that are used on the platform that are meant to be accessed through a nodeport.
The apis need to be configured to be accessible through the .meepctl-repocfg.yaml file
3 new tags added:
- nodeport : nodeport where the rest api can be accessed
- api : the yaml file that contains the rest api
- apihost : boolean that specifies that this service hosts all the swagger ui at <hostname>:nodeport/api

The swagger-ui provides a convenient way to use the TRY-IT-OUT feature to test the api. Note that this may impact currently active scenario. 

The swagger-ui is accessible at <hostname>:30000/api

### CODE REVIEW

**Comment by pastorsx** [comment]
> Please rebase to tip of develop branch.
> Thanks!

rebased just now

File: swagger-ui/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!-- HTML for static distribution bundle build -->
```
Comment (dilallkx) [inline swagger-ui/dist/index.html]: - Move the top-level swagger-ui folder to 'js-apps/meep-swagger-ui'
- Remove dist folder if not required...

File: .meepctl-repocfg.yaml [hunk 2]
```diff
@@ -27,6 +27,9 @@ repo:
       chart: charts/meep-ctrl-engine
       codecov: true
       lint: true
+      nodeport: 30000
+      api: go-apps/meep-ctrl-engine/api/swagger.yaml
+      apihost: true
       docker-data:
         static: bin/meep-frontend
```
[the remark this answers is not in the document]
Comment (pastorsx) [reply .meepctl-repocfg.yaml]: apihost removed
api added umder docker-data... eventually copied under static/api in dockerfile
meep-swagger-ui component created as per suggestions

File: go-apps/meepctl/cmd/dockerize.go [hunk 3]
```diff
@@ -140,6 +143,130 @@ func dockerize(registry string, targetName string, cobraCmd *cobra.Command) {
 				_, _ = utils.ExecuteCmd(cmd, cobraCmd)
 				cmd = exec.Command("cp", "-r", srcDataDir, dstDataDir)
 				_, err = utils.ExecuteCmd(cmd, cobraCmd)
+				//copy bin/api
+				if apiHost {
```
[the remark this answers is not in the document]
Comment (pastorsx) [reply go-apps/meepctl/cmd/dockerize.go]: moved from dockerize.go to build.go

File: swagger-ui/dist/index.html (hunk 1 above)
Comment (pastorsx) [reply swagger-ui/dist/index.html]: done

**Pull request merged**